examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -450,7 +453,14 @@ CHIP_ERROR CHIPCommand::InitializeCommissioner(CommissionerIdentity & identity,
     std::unique_ptr<ChipDeviceCommissioner> commissioner = std::make_unique<ChipDeviceCommissioner>();
     chip::Controller::SetupParams commissionerParams;
 
-    ReturnLogErrorOnFailure(mCredIssuerCmds->SetupDeviceAttestation(commissionerParams, sTrustStore));
+    if (mDacRevocationSetPath.HasValue())
+    {
+        static chip::Credentials::TestDACRevocationDelegateImpl testDacRevocationDelegate;
```

Comment:
So in interactive mode, if `mDacRevocationSetPath` has no value this will just keep using the revocation set from last time?  Trying to understand the intended behavior here.

Reply:
Yes that is the intended behaviour here, keeping it similar to trust store path. Also, factored out this code into similar method as trust store path.

should we honour the argument provided in interactive mode? If yes, I'll do it in follow up.

Reply:
This is quite different from the trust store path behavior.  The behavior for trust store path is:

1. The first time a command is executed in interactive mode with a trust store path argument, create a FileAttestationTrustStore for that path.
2. Any time after that a command is executed with a trust store path argument, use that FileAttestationTrustStore (with the old path, ignoring the new value).
3. Any command executed without a trust store path argument uses `Credentials::GetTestAttestationTrustStore()`. 
4. Some complications around the env var bits, but let's ignore those.

The behavior that is being implemented here, on the other hand is:

1. The first time a command is executed in interactive mode with a trust store path argument, create a TestDACRevocationDelegateImpl for that path.
2. Any time after that a command is executed, use that same TestDACRevocationDelegateImpl no matter what the revocation path for the command was or whether one was present.

Neither behavior is terribly sane, honestly, but they are also different and I am not sure whether the behavior differences are intentional or not.

Reply:
Thanks for the explanation. Yes, they are not exactly similar. May be once we have default revocation set built up, I can align the behaviour to attestation trust store. I have added the comment in the code, about how it behaves. Also raised #34434